bin/recentfilesmanager.js

```diff
@@ -0,0 +1,122 @@
+var Rollbar = require('rollbar')
+var storage = require('electron-json-storage')
+var log = require('electron-log')
+var TRIALMODE = process.env.TRIALMODE === 'true'
+
+var recentFiles = null
+
+const recentKey = process.env.NODE_ENV === 'dev' ? 'recentFilesDev' : 'recentFiles'
+
+module.exports = class RecentFilesManager {
```

Comment:
I don't think `class` will work here (if it does, then no problem) because i don't think this file is being compiled by babel (that's how i've set this project up: the renderer process is compiled, but the main process is not … i know it's not the best)